### PR TITLE
Added important caption for githubteacher

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @rogeryanda
- [ ] @jmedinacsu
- [ ] @js3263827
- [ ] @kayakkat
- [ ] @rjenkins5725
- [ ] @liam09
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @IAmHughes
- [ ] @edelgadojr
- [ ] @Phat6
- [ ] @octowilliam
- [ ] @jrhixson2002
- [ ] @dannyreid
- [ ] @mgthompson2811
- [ ] @anshubhatia
- [ ] @rogeryanda
- [ ] @githubteacher
- [ ] @Phat6
- [ ] @dannyreid
- [ ] @jrhixson2002
- [ ] @js3263827
- [ ] @jmedinacsu
- [ ] @IAmHughes
- [ ] @liam09
- [ ] @rjenkins5725
- [ ] @edelgadojr
- [ ] @rogeryanda
- [ ] @octowilliam
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @rtsinclair
- [ ] @rogeryanda
- [ ] @gmeyer39
- [ ] @rtsinclair
- [ ] @lkelly1
- [ ] @js3263827
- [ ] @githubteacher
- [ ] @githubteacher
- [ ] @kayakkat
- [ ] @jrhixson2002
- [ ] @jrhixson2002
- [ ] @gmeyer39
- [ ] @lgibbens

### PR DESCRIPTION
**Pull request by liam09**
TL;DR - Added important question and changed caption

##Tasks Remaining
-[] Get a Review

Link to issue or service now
This resolves #2 

Leading questions being asked here or something

### CODE REVIEW

File: _posts/0000-01-26-liam09.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Where's the Any Key. Wait, I want a Tab...
```
Comment (jrhixson2002) [inline _posts/0000-01-26-liam09.md:6]: I prefer coke :smiley_cat:

**Review by jrhixson2002 — CHANGES_REQUESTED** [review]
@liam09 this is making me thirsty :tropical_drink:

**Review by jrhixson2002 — APPROVED** [review]
